.rubocop.yml

```diff
@@ -5,6 +5,8 @@ require:
  - rubocop-sorbet
  - rubocop-minitest
  - rubocop-rake
+ # enable this to test handling of failing cops
+ # - ./test/fixtures/custom_cops/duffy
```

Comment:
This is helpful for verifying the behaviour on this PR, but I doubt we'll ever need this again. We should remove these associated configs and the duffy cop before shipping.